cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -511,7 +518,12 @@ private Stream<CliCheckResult> getCheckerHardFailures(List<CliCheckResult> resul
   }
 
   private CliCheckerExecutor getCliCheckerExecutor() {
-    return new CliCheckerExecutor(generateCheckerList(generateCheckerOptions()));
+    Pattern excludedFilesPattern =
+        Strings.isNullOrEmpty(this.excludedFilesRegex)
+            ? null
+            : Pattern.compile(this.excludedFilesRegex);
+    return new CliCheckerExecutor(
+        generateCheckerList(generateCheckerOptions()), excludedFilesPattern);
```

Comment:
As above, I would place the pattern in checker options

Reply:
Done